databricks/koalas/typedef.py

```diff
@@ -17,6 +17,7 @@
 """
 Utilities to deal with types. This is mostly focused on python3.
 """
+
```

Comment:
can you remove this unrelated change?

Reply:
Thanks for pointing out this mistake. 
I'll rebase this PR.